app/models/katello/subscription.rb

```diff
@@ -14,7 +14,7 @@ class Subscription < Katello::Model
 
     def self.subscribable
       joins("LEFT OUTER JOIN #{Katello::SubscriptionProduct.table_name} subprod ON #{self.table_name}.id = subprod.subscription_id")
-        .where("subprod.product_id" => Product.subscribable << nil)
+        .where("subprod.product_id" => (Product.subscribable + [nil]))
```

Comment:
So, I guess the idea is to get `Product.subscribable` to hit the DB before the rest of the query?  What's wrong with keeping it an AR::Relation?

Reply:
In rails 4, you could do:

Object.some_scope <<  Another object
which would cause 'Object.some_scope' to be implicitly converted to an array.  This no longer works with Rails 5.  We hit this quite a few places, this one just snuck in after we did most of the rails 5 fixes.

Reply:
I mean, why not call `.pluck(:id)`?

Reply:
sure, updated